pylookyloo/api.py

```diff
@@ -460,4 +460,10 @@ def get_modules_responses(self, tree_uuid: str) -> Dict[str, Any]:
         :param capture_uuid: UUID of the capture
         '''
         r = self.session.get(urljoin(self.root_url, str(PurePosixPath('json', tree_uuid, 'modules'))))
-        return r.json()
\ No newline at end of file
+        return r.json()
+    
+    def send_mail(self, tree_uuid: str, email: str = '', comment: str = None):
```

Comment:
Can you add typing for what the method returns? After the changes requested in the other PR, it will be either `bool`, or `dict[str, str]`.